shared/django_apps/core/models.py

```diff
@@ -139,6 +139,7 @@ class Languages(models.TextChoices):
     deleted = models.BooleanField(default=False)
     bundle_analysis_enabled = models.BooleanField(default=False, null=True)
     coverage_enabled = models.BooleanField(default=False, null=True)
+    tests_enabled = models.BooleanField(default=False, null=True)
```

Comment:
Can we slightly change this to be `test_analytics_enabled` just to be super clear as to what we're talking about?

Reply:
✅